test/dlc_tests/sanity/quick_checks/test_yml_config.py

```diff
@@ -0,0 +1,57 @@
+import os
+import re
+
+import pytest
+import yaml
+
+from test.test_utils import is_pr_context
+
+
+@pytest.mark.quick_checks
+@pytest.mark.model("N/A")
+@pytest.mark.integration("release_images_training.yml")
+@pytest.mark.skipif(
+    not is_pr_context(),
+    reason="This test is only needed to validate release_images configs in PRs.",
+)
+def test_release_images_training_yml():
+    _release_images_yml_verifier(image_type="training", excluded_image_type="inference")
+
+
+@pytest.mark.quick_checks
+@pytest.mark.model("N/A")
+@pytest.mark.integration("release_images_inference.yml")
+@pytest.mark.skipif(
+    not is_pr_context(),
+    reason="This test is only needed to validate release_images configs in PRs.",
+)
+def test_release_images_inference_yml():
+    _release_images_yml_verifier(image_type="inference", excluded_image_type="training")
```

Comment:
Do you we see a case where we may have more than 2 image types in the future? If yes, then we should make excluded image type parameter a list of matching patterns. 
Alternatively, make _release_images_yml_verifier one parameter-function with image_type as parameter; then convert yml into dictionary and for each entry search and validate that each entry matches the key structure of [framework, version, customer_type, arch_type, image_type].